### DISCUSSION TITLE
zone2sql import says successfully but not importing

### DISCUSSION THREAD

**Discussion by snipecoder** [PowerDNS/pdns#14181] [Q&A] [answered]
Hi

I added a simple txt file contains the following records:
x.x.x.in-addr.arpa.txt:
1	600	IN	PTR	mta59-1.sub59.x.com.
10	600	IN	PTR	mta59-10.sub59.x.com.
100	600	IN	PTR	mta59-100.sub59.x.com.

But that not importing - no errors. I can see the log:
zone2sql --gmysql --zone=/root/ptr/x.x.x.in-addr.arpa.txt --zone-name=x.x.x.in-addr.arpa:
insert into domains (name,type) values ('x.x.x.in-addr.arpa','NATIVE');
insert into records (domain_id, name, type,content,ttl,prio,disabled) select id ,'1.x.x.x.in-addr.arpa', 'PTR', 'mta59-1.sub59.x.com', 600, 0, 0 from domains where name='x.x.x.in-addr.arpa';
insert into records (domain_id, name, type,content,ttl,prio,disabled) select id ,'10.x.x.x.in-addr.arpa', 'PTR', 'mta59-10.sub59.x.com', 600, 0, 0 from domains where name='x.x.x.in-addr.arpa';
1 domains were fully parsed, containing 255 records

The domain is there, all seems okay, but not importing - any idea?

**Comment by franklouwers** [comment, marked as answer]
https://doc.powerdns.com/authoritative/manpages/zone2sql.1.html

> zone2sql parses BIND named.conf files and zonefiles and outputs SQL on standard out, which can then be fed to your database.

This means you'll need to feed the sql to the database yourself, as it says on the first line of the manpage for that tool.

**Reply by snipecoder** [reply]
Oh my gosh, I am so stupid, thank you.
